README.md

```diff
@@ -3,6 +3,10 @@
 This is a compiled list of public talks by Zalando Tech: meetup presentations, recorded conference talks, slides, etc.
 We try to keep the list up-to-date --- please open a Pull Request to add a public presentation.
 
+## 2025
```

Comment:
Thanks for the additions. Can you please drop the github handles? It's inconsistent with the existing content. Names are in titles of the videos, slide decks, and this PR.

Reply:
done